docs/builds/guides/frameworks/angular.md

```diff
@@ -299,6 +331,171 @@ Then, add it in the `angular.json` configuration file.
 }
 ```
 
+## Accessing the CKEditor API from an Angular Component
+
+The **ckeditor** component will provide all the functionality needed for most use cases.  In 
```

Comment:
Please use:

```
`<ckeditor>` component
```

Reply:
The same applies to all other bold use – please use backticks instead.

Reply:
@AlanObject, could you fix these small issues here?